sechub-pds-solutions/cloc/helm/pds-cloc/values.yaml

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: MIT
+
+# This is just a sample values file.
+
+replicaCount: 1
+
+image:
+    registry: "ghcr.io/mercedes-benz/sechub/pds-cloc:latest"
+    tag: "latest"
+
+pds:
+    startMode: localserver
+    config:
+        execute:
+            queueMax: 10
+            workerThreadCount: 1
+    heartbeatLogging: "true"
+    autoclean: true
```

Comment:
`pds.autoclean` is never used in the templates.
1) remove it here - or
2) add it to the templates/deployment.yaml file